pilot/proxy/envoy/route.go

```diff
@@ -376,7 +376,11 @@ func buildTCPRoute(cluster *Cluster, addresses []string) *TCPRoute {
 	}
 	sort.Sort(sort.StringSlice(addresses))
 	for _, addr := range addresses {
-		route.DestinationIPList = append(route.DestinationIPList, addr+"/32")
+		tcpRouteAddr := addr
```

Comment:
nit: this could be a bit shorter if you append to `addr` directly and avoid the extra var declaration. 

```golang
if !strings.Contains(addr, "/") {
       	addr = addr + "/32"
}
route.DestinationIPList = append(route.DestinationIPList, addr)
```

Reply:
The extra var is for documentation - to explain that I convert the input address to the tcp route address field of envoy.